src/main/java/org/mifos/pheeBillPay/api/implementation/BillPaymentsController.java

```diff
@@ -20,6 +20,12 @@ public class BillPaymentsController implements BillPaymentsApi {
     public ResponseEntity<BillInquiryResponseDTO> billPayments(String tenantId, String correlationId, String callbackURL, String payerFspId,
             BillPaymentsReqDTO body) throws ExecutionException {
         BillInquiryResponseDTO billInquiryResponseDTO = new BillInquiryResponseDTO();
+        if(tenantId == null || correlationId == null || callbackURL == null ||
```

Comment:
write a function to do this comparison, it is not readable to see so many conditions in between the business logic